properties.schema

```diff
@@ -83,10 +83,10 @@
                 "_routeToLocation": {
                   "type": "string",
                   "title": "Link",
-                  "help": "When set to a valid Adapt element, the logo image will link to the specified location. Valid values include a page (e.g. co-100) or a menu (e.g. course)",
+                  "help": "When set to a valid Adapt element, the logo image will link to the specified location. Valid values include a page (e.g. co-100) or a menu (e.g. course). @course can be used to always returns to the main menu of the course",
                   "required": false,
                   "validators": [],
-                  "default": ""
+                  "default": "@course"
```

Comment:
Remove link from default property in schema
```suggestion
                  "default": ""
```

Reply:
Why remove the default value that the majority of people will want to use?

Reply:
Sorry, I forgot that there wasn't a `_enableRouting` attribute attached to this, so, when blank, is it being just a logo rather than a navigation button.

In that case, I'm happy to remove the value.